Спільні підкаталоги: davidm-lua2c-c5b239d.orig/examples-lua та davidm-lua2c-c5b239d/examples-lua
Спільні підкаталоги: davidm-lua2c-c5b239d.orig/lib та davidm-lua2c-c5b239d/lib
diff -u davidm-lua2c-c5b239d.orig/lua2c.lua davidm-lua2c-c5b239d/lua2c.lua
--- davidm-lua2c-c5b239d.orig/lua2c.lua	2010-08-31 18:23:18.000000000 +0300
+++ davidm-lua2c-c5b239d/lua2c.lua	2013-01-11 11:12:45.676285966 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env lua5.1
 -- lua2c.lua - Driver for lua2c - converts Lua 5.1 source to C code.
 --
 -- STATUS:
